gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/config/gravitee.yml

```
@@ -346,15 +346,15 @@ repositories:
   #      type:                      # Type of the truststore, supports jks, pem, pkcs12 (when sslEnabled is true, default null)
   #      password:                  # Truststore password (when sslEnabled is true, default null)
   gateway:
-    type: ${repositories.management.type}
+    type: ${oauth2.type:${repositories.management.type}}
```

```suggestion
    type: ${repositories.oauth2.type}
```
does this one will fallback to repositories.oauth2.type, which is configured to fallcack to oauth2.type